trunk/SUAVE/Methods/Weights/Buildups/eHelicopter/wing.py

```diff
@@ -0,0 +1,248 @@
+# wing.py
+#
+# Created: Jun 2017, J. Smart
+
+#-------------------------------------------------------------------------------
+# Imports
+#-------------------------------------------------------------------------------
+
+from SUAVE.Core import Units
+from SUAVE.Attributes import Solids
+import numpy as np
+
+#-------------------------------------------------------------------------------
+# Wing
+#-------------------------------------------------------------------------------
+
+def wing(MTOW, wingspan, chord, thicknessToChord, wingletFraction,
+            liftFraction, xMotor, maxThrust):
+    """weight = SUAVE.Methods.Weights.Correlations.eHelicopter.wing(
+            MTOW,
+            wingspan,
+            chord,
+            thicknessToChord,
+            wingletFraction,
+            liftFraction,
+            xMotor,
+            maxThrust
+        )
+
+        Calculates the structural mass of a wing for an eVTOL vehicle based on
+        assumption of NACA airfoil wing, an assumed L/D, cm/cl, and structural
+        geometry.
+
+        Intended for use with the following SUAVE vehicle types, but may be used
+        elsewhere:
+
+            eTiltwing
+            eTiltrotor
+            eStopped_Rotor
+
+        Originally written as part of an AA 290 project intended for trade study
+        of the above vehicle types plus an eHelicopter.
+
+        Inputs:
+
+            MTOW:               Maximum TO Weight       [N]
+            wingspan:           Wingspan                [m]
+            chord:              Wing Chord              [m]
+            wingletFraction:    Winglet Length/Wingspan [Unitless]
+            thicknessToChord:   Wing t/c Ratio          [Unitless]
+            liftFraction:       Fraction of Total Lift  [Unitless]
+            xMotor:             Motor Span Fractions    [Unitless]
+            maxThrust:          Maximum Motor Thrust    [N]
+
+        Outputs:
+
+            weight:             Wing Mass               [kg]
+    """
+
+    N       = 10            # Number of spanwise points
```

Comment:
Allow these assumptions to be inputted or defaulted.

Reply:
Changed to function input with default value.